commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AuditLogging.java

```diff
@@ -0,0 +1,126 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+/**
+ * A slightly more convenient way to create commands for Audit Logging. All parameters in this class's methods must be
+ * specified.
+ */
+public final class AuditLogging {
+
+    private AuditLogging() {}
+
+    public static AuditLogDestinationHandler destinationHandler() {
+        return AuditLogDestinationHandler.INSTANCE;
+    }
+
+    public static AuditLogger auditLogger() {
+        return AuditLogger.INSTANCE;
+    }
+
+    /**
+     * Configures the destinations where should be the auditable events sent to.
+     */
```

Comment:
Comments like this shouldn't be on the classes, but on the methods that return the `INSTANCE`. Users won't typically care about these classes, but they will always care about the methods.